capstone/src/main/java/com/google/univiz/api/VisResource.java

```diff
@@ -1,7 +1,40 @@
 package com.google.univiz.api;
 
+import com.google.univiz.CollegeData;
+import com.google.univiz.api.dataviz.Deadline;
+import com.google.univiz.api.dataviz.Timeline;
+import com.google.univiz.api.dataviz.TuitionStats;
+import java.util.List;
+
 /**
  * The VisResource interface is responsible for all data management/processing specific for the vis
  * feature of the UniViz webapp.
  */
-public interface VisResource {}
+public interface VisResource {
+
+  /**
+   * For every college the user selected, obtain the raw statistics and compute summary statistics
+   * on tuition-related data.
+   *
+   * @param colleges the raw data for each of the user's selected colleges
+   * @return the list of statistics per college.
+   */
+  List<TuitionStats> getTuitionStats(List<CollegeData> colleges);
```

Comment:
Is the client going to have to supply all of the college data back to the methods in this resource? Or can the client simply supply the list of CollegeIds? 

One corner case I can think of is the first load, when the client doesn't have any access to collegedata...

Reply:
I think the methods can still behave as intended even if `CollegeIds` are passed. I will change the signature for these methods.

Reply:
+1 to passing in CollegeIds only.